src/components/Table/ListTable/TableAction/QuickFilter.vue

```diff
@@ -88,6 +92,10 @@ export default {
       }
     }
   },
+  mounted() {
+    console.log(this.filters)
+    console.log(this.summary)
+  },
   methods: {
     async getCount(item) {
       if (item.count || item.count === 0) {
```

Comment:
I'm sorry, but without having access to the code you want me to analyze, I cannot perform the necessary checks. Please share the text or the relevant part of the code that needs analysis, and I will do my best to review it.